packages/node_modules/@ciscospark/plugin-avatar/src/avatar.js

```diff
@@ -96,7 +94,7 @@ const Avatar = SparkPlugin.extend({
     })
       .then((res) => {
         // invalidate user's cached avatar
-        this.store.remove(this.spark.device.userId);
+        this.store.remove({uuid: this.spark.device.userId});
         return res.url;
```

Comment:
remove was expecting an "item" object, not just a uuid. This prevented cleanup of all URLs associated with uuid